reframe/core/modules.py

```diff
@@ -203,6 +204,27 @@ def is_module_loaded(self, name):
     def _is_module_loaded(self, name):
         return self._backend.is_module_loaded(Module(name))
 
+    def load_mapping(self, mapping):
+        key, values = mapping.strip().strip(':').split(':')
+        self.module_map[key.strip()] = list(
+            OrderedDict.fromkeys(values.split()))
+
+    def load_mapping_from_file(self, filename):
+        """Update the internal module mapping from a file."""
+
+        with open(filename, 'r') as fp:
+            for lineno, line in enumerate(fp):
+                line = line.strip().split('#')[0]
```

Comment:
Because if the commented line has a space before, `split('#')` will return a list with the first item an space `[' ', '#...']` and I preffered an empty line, just to skip it the same way that an empty line in the file is skipped.